lib/structures/Message.js

```diff
@@ -294,6 +295,15 @@ class Message extends Base {
         if(data.stickers !== undefined) {
```

Comment:
Q: In what cases do `stickers` get sent over `stickerItems` now? Could be doing something wrong, but interestingly I couldn't see a single instance where `stickers` was present when I was listening for raw MESSAGE_CREATEs containing stickers (even though it's just marked as deprecated in the docs).

Reply:
They never do. Discord marked the property as deprecated in the docs, however in reality it was actually removed and replaced with `sticker_items`. In my original commits, I simply changed the Message#stickers property to use `sticker_items`, but Bsian felt it was necessary to deprecate the old value and add `stickerItems` to keep things consistent. 👍🏻

Reply:
I did just confirm - old messages before the change still use `stickers` instead of `sticker_items` (as such, this change would break backwards compatibility with those messages)

Reply:
Ahh that makes sense, thanks for clarifying.